plugins/discourse-narrative-bot/plugin.rb

```diff
@@ -194,6 +194,15 @@ def fetch_avatar(user)
     return if topic.blank?
 
     first_post = topic.ordered_posts.first
+
+    notification = Notification.where(topic_id: topic.id, post_number: first_post.post_number).first
+    if (notification.present?)
```

Comment:
Surprised rubocop didn't pick that one

```suggestion
    if notification.present?
```